tests/Maker/MakeEntityTest.php

```diff
@@ -44,18 +44,12 @@ private function createMakeEntityTestForMercure(): MakerTestDetails
             ;
         }
 
-        // @legacy - MakeEntity uses ux-turbo-mercure (archived), it needs to use ux-turbo (mercure built in) for Symfony 7.0
-        if ('7.0.x-dev' === $_SERVER['SYMFONY_VERSION']) {
-            return $this->createMakerTest()
-                ->skipTest('symfony/ux-turbo-mercure is not supported on Symfony 7.')
-            ;
-        }
-
         return $this->createMakeEntityTest()
             ->preRun(function (MakerTestRunner $runner) {
                 // installed manually later so that the compatibility check can run first
-                $runner->runProcess('composer require symfony/ux-turbo-mercure');
+                $runner->runProcess('composer require symfony/ux-turbo');
             })
+            ->addExtraDependencies('mercure', 'twig')
```

Comment:
`ux-turbo` installs `stimulus-bundle` which requires `twig`. But the twig recipe doesn't get generated in the `tmp/cache` project. adding it as an extra depend fixes that.

`mercure` is needed obviously to pub/sub changes